### PR TITLE
Update inline-message-translation-teams.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5393
Changed wording.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/inline-message-translation-teams.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Turn on the policy by using the [Set-CsTeamsMessagingPolicy](https://docs.micros
 
 ## Use the Microsoft Teams admin center to turn on inline message translation
 
-In the **Microsoft Teams admin center**, select **Messaging Policies** from the left navigation, then either create a new policy or edit an existing policy, and set the **Allow users to translate messages** option to **On**.
+In the **Microsoft Teams admin center**, select **Messaging Policies** from the left navigation, then either create a new policy or edit an existing policy, and set the **Translate messages** option to **On**.
 
 > [!NOTE]
-> The service does the translation and delivers it to the client with no effect on the content captured in the compliance records. To learn more about translation, see [What is Microsoft Translator?](https://docs.microsoft.com/azure/cognitive-services/translator/translator-info-overview).
\ No newline at end of file
+> The service does the translation and delivers it to the client with no effect on the content captured in the compliance records. To learn more about translation, see [What is Microsoft Translator?](https://docs.microsoft.com/azure/cognitive-services/translator/translator-info-overview).
```
Comment (JohanFreelancer9) [inline Teams/inline-message-translation-teams.md:46]:
```suggestion
> The service does the translation and delivers it to the client with no effect on the content captured in the compliance records. To learn more about translation, see [What is Microsoft Translator?](https://docs.microsoft.com/azure/cognitive-services/translator/translator-info-overview)
```

Comment (JohanFreelancer9) [reply Teams/inline-message-translation-teams.md:46]: Please remove the period at the end of the Note.

**Comment by scanum** [comment]
@chuckedmonson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**